modules/manage/pages/manage-xdcr/incoming-xdcr-replications.adoc

```diff
@@ -0,0 +1,63 @@
+= Incoming Replications
+:description: pass:q[Incoming replications details for a target cluster in XDCR are listed on the UI and can be retrieved using the REST API.]
+:page-toclevels: 2
+
+[abstract]
+{description}
+
+When a replication is created, the target cluster is identified as such.
+
+A cluster is identified as a target cluster in XDCR when it receives replicated data from a remote cluster.
+Then the replicating remote cluster is identified as a source cluster.
+
+[#view-incoming-replications]
+== View Incoming Replications
+
+On the XDCR UI, if the cluster is a target for XDCR replications from remote clusters, the Incoming Replications section lists the incoming replications, grouped by the remote cluster name.
+Expand *Incoming Replications* to see the details of any incoming replications created on the remote cluster.
+
+There may not be any replications listed if the remote cluster has not yet created a replication.
+However, the remote cluster name is listed because the remote cluster has added this cluster's connection details to its XDCR cluster references.
+
+image::manage-xdcr/incoming-replications-on-target.png[,800,align=left]
+
+The information for the above incoming replications is retrieved using the REST API.
+See xref:rest-api:rest-xdcr-list-incoming-replications.adoc#complete-response[Cluster with replication specifications - a full example].
```

Comment:
I think that lines 24 and 25 should be combined.

Suggesting:
`To see the same Incoming Replications info retrieved using the REST API, see xref:rest-api:rest-xdcr-list-incoming-replications.adoc#complete-response[Cluster with replication specifications - a full example].`